src/QsCompiler/QirGeneration/BondSchemas/Extensions.cs

```diff
@@ -1,8 +1,6 @@
 ﻿// Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
-#nullable enable
```

Comment:
Why are we removing this?

Reply:
This has been moved to the .csproj file and became applicable to the whole project.